tests/bluetooth/ctrl_isoal/src/main.c

```diff
@@ -20,1517 +20,7747 @@
 /* Include the DUT */
 #include "ll_sw/isoal.c"
 
-/*
- * Note: sdu_interval unit is uS, iso_interval is a multiple of 1.25mS
- *
- * pdus_per_sdu = burst_number * (sdu_interval / (iso_interval * 1250));
- *
- */
+/* #define DEBUG_TEST			(1) */
+/* #define DEBUG_TRACE			(1) */
```

Comment:
Should these be kept or removed?

Reply:
Remove

Reply:
If the debugging functionality and prints are to be enabled, those will need to be defined. The previous unit tests also had it, however if there is a standard approach to this that I am not following, do let me know.

Reply:
I am fine with having DEBUG_TEST and DEBUG_TRACE, but they are commented out here. IMO it's general bad practice to have commented code in the tree (especially without any documentation as to _why_). 

Is there are reason why you are not using something like the `CONFIG_BT_TESTING`?

Reply:
Thanks for the suggestion of `CONFIG_BT_TESTING`, but I don't think that's quite right as these aren't test commands.

It would be possible to link this to a Debug build of the unit tests if such a thing exists. However I don't know if that exists or if so how it is configured and used.